app/models/live_metric.rb

```diff
@@ -0,0 +1,112 @@
+class LiveMetric < ActsAsArModel
+  set_columns_hash(:timestamp => :datetime)
+
+  class LiveMetricError < RuntimeError; end
+
+  def self.where(*)
+    raise LiveMetricError, "where method is not supported on LiveMetric"
```

Comment:
can't this be handled by ruby, or I'm missing something in the context of the change? (NoMethoderror or NotImplementedError, for example)